src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation/positive/service-def-valid-parameters.json

```diff
@@ -0,0 +1,86 @@
+﻿{
```

Comment:
There are 2 rules in play here:
1. Operation parameters MUST not include the subscriptionId and ApiVersion parameters 
2. ServiceDefinition global parameters MUST have a definition for subscriptionId (as long as it is being referenced in any of the url params) and apiversion parameters

This json has a URL reference to subscriptionId and a corresponding definition in the global parameters section for both subscriptionId and apiversion